storybook/src/utils/design-tokens/FontDesignToken.tsx

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
This 'utils' folder is not a folder of Storybook utils, but a chapter visible to users of our Storybook instance.

We've had this issue before, maybe we should consider renaming. In _meldingen_ we would do something like `_utils` and `_styles` for the non-public facing stuff. And maybe we should rename `utils` to `utilities`, to reflect the Storybook chapter name?